samples/scaleout/senderside-and-distributor/Core_6/Worker.Handlers/OrderValidatedCommandHandler.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Reflection;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Logging;
+
+public class OrderValidatedCommandHandler :
+    IHandleMessages<OrderValidated>
+{
+    static ILog log = LogManager.GetLogger<OrderValidatedCommandHandler>();
+
+    public async Task Handle(OrderValidated message, IMessageHandlerContext context)
+    {
+        log.Info($"Order {message.OrderId} validated. Sending notification to the customer.");
+
+        await Task.Delay(1000).ConfigureAwait(false);
+
+        var placed = new OrderPlaced
+        {
+            OrderId = message.OrderId,
+            WorkerName = Assembly.GetEntryAssembly().GetName().Name
+        };
+
+        var options = new SendOptions();
+        options.SetDestination(message.Sender);
+        await context.Send(placed, options)
```

Comment:
renamed to `PlaceOrderResponse`. Nope. We send it after a bunch of SendLocal's which override the ReplyTo header.